src/shared/components/VideoTileBase/VideoTileBase.styles.tsx

```diff
@@ -5,13 +5,19 @@ import { Link } from 'react-router-dom'
 
 import { colors, media, sizes, square, transitions, typography, zIndex } from '@/shared/theme'
 
+import { TileSize } from './VideoTileBase'
```

Comment:
Let's not do this - it will create a circular dependency between `VideoTileBase.styles` and `VideoTileBase`. Let's either define `TileSize` here or move types to a separate file